### ISSUE TITLE
Style Analysis GUI

### ISSUE THREAD

**Issue by ghost** [NLP-Suite/NLP-Suite#1277]
I am trying to run the Style Analysis GUI, which is not opening for me.

Here's a screenshot of the NLP Main GUI with the error that shows up:
<img width="1332" alt="PNG image" src="https://github.com/NLP-Suite/NLP-Suite/assets/143128568/6068d227-69af-4b3c-8d1d-ccf74327c219">

Here is the terminal code for this error:
Last login: Mon Oct 30 10:45:24 on ttys000
/Users/keerthyrangan/Documents/Sociology\ 446W/NLP-Suite-current-stable/setup_Mac/run_NLP-Suite.command ; exit;
(base) keerthyrangan@Keerthys-MacBook-Air ~ % /Users/keerthyrangan/Documents/Sociology\ 446W/NLP-Suite-current-stable/setup_Mac/run_NLP-Suite.command ; exit;

--------------- platform.machine()

---------------  arm64

--------------- output

---------------  absl-py==1.4.0
annotated-types==0.5.0
appnope==0.1.3
asttokens==2.2.1
astunparse==1.6.3
autocorrect==2.3.0
backcall==0.2.0
bert-extractive-summarizer==0.10.1
blis==0.7.10
cachetools==5.3.1
catalogue==2.0.9
certifi==2023.7.22
cffi==1.15.1
chardet==5.2.0
charset-normalizer==3.2.0
click==8.1.7
confection==0.1.1
conllu==4.5.3
contextualSpellCheck==0.4.3
contourpy==1.1.0
cryptography==41.0.3
cycler==0.11.0
cymem==2.0.7
Cython==3.0.0
decorator==5.1.1
editdistance==0.6.0
emoji==2.8.0
en-core-web-sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.6.0/en_core_web_sm-3.6.0-py3-none-any.whl#sha256=83276fc78a70045627144786b52e1f2728ad5e29e5e43916ec37ea9c26a11212
et-xmlfile==1.1.0
exceptiongroup==1.1.3
executing==1.2.0
filelock==3.12.2
flatbuffers==23.5.26
fonttools==4.42.1
fsspec==2023.6.0
funcy==2.0
future==0.18.3
fuzzywuzzy==0.18.0
gast==0.4.0
gensim==4.3.2
geographiclib==1.52
geopy==2.1.0
google-auth==2.22.0
google-auth-oauthlib==1.0.0
google-pasta==0.2.0
googlemaps==4.4.7
grpcio==1.57.0
h5py==3.9.0
huggingface-hub==0.16.4
idna==3.4
importlib-metadata==6.8.0
importlib-resources==6.0.1
iniconfig==2.0.0
ipython==8.12.2
isodate==0.6.1
jedi==0.19.0
Jinja2==3.1.2
joblib==1.3.2
kaleido==0.2.1
keras==2.13.1
kiwisolver==1.4.5
langcodes==3.3.0
langdetect==1.0.7
langid==1.1.6
Levenshtein==0.21.1
libclang==16.0.6
lxml==4.6.5
Markdown==3.4.4
MarkupSafe==2.1.3
matplotlib==3.3.4
matplotlib-inline==0.1.6
mlconjug==3.4.0
mpmath==1.3.0
murmurhash==1.0.9
networkx==3.1
nltk==3.8.1
numexpr==2.8.5
numpy==1.24.3
oauthlib==3.2.2
openpyxl==3.0.10
opt-einsum==3.3.0
packaging==23.1
pandas==2.0.3
parso==0.8.3
pathy==0.10.2
pdfminer.six==20221105
pexpect==4.8.0
pickleshare==0.7.5
Pillow==9.5.0
plotly==5.16.1
pluggy==1.2.0
preshed==3.0.8
prompt-toolkit==3.0.39
protobuf==3.20.0
psutil==5.9.5
ptyprocess==0.7.0
pure-eval==0.2.2
pyasn1==0.5.0
pyasn1-modules==0.3.0
pycorenlp==0.3.0
pycparser==2.21
pydantic==2.3.0
pydantic_core==2.6.3
pygit2==1.12.2
Pygments==2.16.1
pyLDAvis==2.1.2
pyparsing==3.0.9
pyphen==0.14.0
pyspellchecker==0.7.2
pytest==7.4.0
python-dateutil==2.8.2
python-docx==0.8.11
python-Levenshtein==0.21.1
pytz==2023.3
pywsd==1.2.5
PyYAML==6.0.1
rapidfuzz==3.2.0
rdflib==7.0.0
regex==2023.8.8
requests==2.31.0
requests-oauthlib==1.3.1
rsa==4.9
safetensors==0.3.3
scikit-learn==1.3.0
scipy==1.10.1
sentence-transformers==2.2.2
sentencepiece==0.1.99
simplekml==1.3.6
six==1.16.0
smart-open==6.3.0
spacy==3.6.1
spacy-langdetect==0.1.2
spacy-legacy==3.0.12
spacy-loggers==1.0.4
spacytextblob==4.0.0
SPARQLWrapper==2.0.0
srsly==2.4.7
stack-data==0.6.2
stanfordcorenlp==3.9.1.1
stanza==1.5.0
striprtf==0.0.26
summarizer==0.0.7
sympy==1.12
tenacity==8.2.3
tensorboard==2.13.0
tensorboard-data-server==0.7.1
tensorflow==2.13.0
tensorflow-estimator==2.13.0
tensorflow-macos==2.13.0
termcolor==2.3.0
textblob==0.15.3
textstat==0.7.3
thinc==8.1.12
threadpoolctl==3.2.0
tkcolorpicker==2.1.3
tokenizers==0.13.3
tomli==2.0.1
torch==2.0.1
torchvision==0.15.2
tqdm==4.55.2
traitlets==5.9.0
transformers==4.32.0
twython==3.9.1
typer==0.9.0
typing_extensions==4.7.1
tzdata==2023.3
unicode==2.9
Unidecode==1.3.6
urllib3==1.26.16
wasabi==1.1.2
wcwidth==0.2.6
Werkzeug==2.3.7
wn==0.0.23
wordcloud==1.8.2.2
wrapt==1.15.0
zipp==3.16.2

No module named 'seaborn'

When I tried to install seaborn, it was successfully installed, but the same error was given when I ran NLP Suite again.

**Comment by bianshuyang** [comment]
I do not believe you install it under the correct environment.
